sdk/master_board_sdk/example/example.cpp

```diff
@@ -8,8 +8,7 @@
 #include "master_board_sdk/master_board_interface.h"
 #include "master_board_sdk/defines.h"
 
-#undef N_SLAVES_CONTROLED
-#define N_SLAVES_CONTROLED 2
+#define N_SLAVES_CONTROLED 1
```

Comment:
This looks like a change done during development. I don't mind if we reduce by default the #slaves in the example from 2 -> 1. (Same for python example below.)